### PR TITLE
DS-3136 Add support for higher dim Q tables in SumRows/Cols

### PR DESCRIPTION
**Pull request by jrwishart**
Added ability for SumRows and SumColumns to handle three to five dimensional arrays that occur when some multiple variable sets are used in some Tables.

### CODE REVIEW

**Comment by jrwishart** [comment]
Test documents available https://australiaeast.displayr.com/Dashboard?project_id=-849292#page=bd6b3c94-ec3e-4b0f-a4d0-1a702adf9d70 and https://australiaeast.displayr.com/Dashboard?project_id=-849851#page=58791ce4-5611-4d78-b9aa-af9d940706aa

File: R/sumrows.R [hunk 1]
```diff
@@ -129,14 +130,16 @@ getColumnNames <- function(x)
         return(symbol)
 }
 
-sumRows <- function(x, remove.missing)
+sumRows <- function(x, remove.missing, remove.columns)
 {
     x.names <- rowNames(x)
     # 2D Table with Multiple statistics is stored as a 3d array
```
[the remark this answers is not in the document]
Comment (jrwishart) [reply R/sumrows.R]: Fixed in https://github.com/Displayr/verbs/pull/6/commits/784f6cf22c3aa002fae3145fc8d9a6fe11f70748

File: DESCRIPTION [hunk 2]
```diff
@@ -27,5 +27,5 @@ Remotes:
     Displayr/flipU
 RoxygenNote: 7.1.1
 Depends: 
-    R (>= 2.10)
+    R (>= 3.50)
```
[the remark this answers is not in the document]
Comment (jrwishart) [reply DESCRIPTION]: Thanks. Fixed in https://github.com/Displayr/verbs/pull/6/commits/784f6cf22c3aa002fae3145fc8d9a6fe11f70748

File: R/sumcolumns.R [hunk 3]
```diff
@@ -34,20 +34,22 @@ SumColumns <- function(...,
     x <- list(...)
     x <- addSymbolAttributeIfPossible(calling.arguments[[2L]], x)
     n.inputs <- length(x)
-    single.QTable.with.multiple.stats <- isQTable(x[[1L]]) && length(dim(x[[1L]])) == 3L
-    three.dim.array <- n.inputs == 1L && single.QTable.with.multiple.stats
     if (n.inputs == 1L)
     {
+        single.higher.dim.array <- n.inputs == 1L && isQTable(x[[1L]]) && length(dim(x[[1L]])) > 2L
```
[the remark this answers is not in the document]
Comment (jrwishart) [reply R/sumcolumns.R]: Thanks. Fixed in https://github.com/Displayr/verbs/pull/6/commits/784f6cf22c3aa002fae3145fc8d9a6fe11f70748

**Comment by mwmclean** [comment]
Closing this in favour of #7

**Pull request closed without merging**